securedrop/tests/functional/test_source.py

```diff
@@ -17,3 +18,16 @@ def test_lookup_codename_hint(self):
         self._source_chooses_to_login()
         self._source_proceeds_to_login()
         self._source_sees_no_codename()
+
+
+class TestDownloadKey(
+        functional_test.FunctionalTest,
+        journalist_navigation_steps.JournalistNavigationStepsMixin):
+
+    def test_journalist_key_from_source_interface(self):
+        data = self.return_downloaded_content(self.source_location +
+                                              "/journalist-key", None)
+
+        if six.PY3:
+            data = data.decode('utf-8')
+        assert "BEGIN PGP PUBLIC KEY BLOCK" in data
```

Comment:
:+1: this should properly assess success/failure: journalist key served was an empty file

Reply:
@emkll - do you mean you got an empty file testing this PR, or 0.13.0 returns an empty file for you? I was seeing a 404 on 0.13.0.

Reply:
Sorry for the confusion:
* In dev env, was getting an empty key (and an error in the logs)
* In staging VMs, was getting 404